org.eclipse.scanning.event/src/org/eclipse/scanning/event/Constants.java

```diff
@@ -17,15 +17,16 @@
 	 * Frequency in ms, default 2000. Set org.eclipse.scanning.event.heartbeat.freq system property to change this time.
 	 */
 	private static long NOTIFICATION_FREQUENCY = 2000;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "NOTIFICATION_FREQUENCY" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)